src/and-cli-install.ts

```diff
@@ -89,31 +84,35 @@ require("./command-runner").run(async () => {
         return true;
     };
 
-    const _installAndCliGlobally = () => {
+    const _installAndCliGlobally = (): void => {
         const cmd = install.cmd(CLI_NAME);
-        echo.message(
+        Echo.message(
             `Installing ${CLI_NAME} as global npm tool... (via ${cmd})`
         );
 
         if (!_execAndExitIfErrored(cmd)) {
             return;
         }
 
-        echo.success(`Successfully installed ${CLI_NAME} globally`);
-        echo.newLine();
+        Echo.success(`Successfully installed ${CLI_NAME} globally`);
+        Echo.newLine();
     };
 
-    const _installDevAlias = () => {
+    const _installDevAlias = (): void => {
```

Comment:
👍 good attention to detail on the install command needing to point to dist. -- in reading this again, I'm thinking we could update this function (in a later PR) to support passing the suffix (ie. dev, etc...). as a maintainer, I keep aliases that point to each of the contributors of this project (ie. `and-cli-{github-username}`). Wondering if the workspace command could also setup/sync aliases too.

Reply:
That would be awesome! I'll add an issue to document this